components/bl-signature-pad-component/component.json

```diff
@@ -0,0 +1,177 @@
+{
+  "id": "c_e4f5716e8bc3ae951a1a1dadd59acd75",
+  "name": "signature-pad",
+  "description": "",
+  "showInToolbox": true,
+  "faIcon": "pencil-ruler",
```

Comment:
Can we change this default icon to some special?

Reply:
Yes, we can. I will change it to "file-signature"